app/controllers/Application.java

```diff
@@ -144,6 +144,22 @@ public static Result dashboard() {
   }
 
   /**
+   * Returns most recent flowExecId that is like value which can use % and _ SQL wild cards.
+   */
+  private static String bestMatchForFlowExecIdLikeValue(String value) {
+    List<AppResult> matches = AppResult.find
+            .select(AppResult.TABLE.FLOW_EXEC_ID)
+            .where().like(AppResult.TABLE.FLOW_DEF_ID, value)
```

Comment:
Maybe I'm misunderstanding but as far as I can tell `setMaxRows(1)` should take care of this issue.